test/libsolidity/smtCheckerTests/array_members/push_overflow_2_safe_no_overflow_assumption.sol

```diff
@@ -13,4 +13,3 @@ contract C {
 // SMTEngine: all
 // SMTIgnoreOS: macos
 // ----
-// Info 1180: Contract invariant(s) for :C:\n(x.length >= 0)\n
```

Comment:
Is that non-determinism too?

Reply:
If it happens on all OSs it should be fine.